client/src/node_api/mqtt/error.rs

```diff
@@ -0,0 +1,20 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+/// MQTT related errors.
+#[derive(Debug, thiserror::Error)]
+#[allow(clippy::large_enum_variant)]
+pub enum Error {
+    /// Client error.
+    #[error("client error {0}")]
+    Client(#[from] rumqttc::ClientError),
+    /// Connection not found, all nodes have their MQTT plugin disabled.
+    #[error("connection not found, all nodes have their MQTT plugin disabled")]
```

Comment:
Let's remove this, because we don't even check that
```suggestion
    /// Connection not found
    #[error("connection not found")]
```

Reply:
Done